externs/olcsx.js

```diff
@@ -5,6 +5,45 @@
  */
 var olcsx;
 
+
+/**
+ * @typedef {{
+ *   map: (!ol.Map),
+ *   target: (Element|string|undefined),
+ *   createSynchronizers: (undefined|function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)
```

Comment:
For example [here](https://github.com/openlayers/ol3/blob/master/externs/olx.js#L6117) and [here](https://github.com/openlayers/ol3/blob/master/externs/olx.js#L6131).